user-guide/Basic_Functionality/Visio/tutorials/Using_Visual_Overview_to_switch_spectrum_presets.md

```diff
@@ -0,0 +1,195 @@
+---
+uid: Using_Visual_Overview_to_switch_spectrum_presets
+---
+
+# Using Visual Overview to switch spectrum presets
+
+This tutorial explains how to start from a blank Visio file and add a spectrum control and several buttons to switch between presets.
+
```

Comment:
Which DataMiner version did you use for this tutorial? We should add this here, so that users with different DataMiner versions will know why what they see is different.

Reply:
Version added. This should be identical for all currently supported DataMiner versions, but in this way it's also future-proof if this should change at some point.